test/validation/api/crypto/test_vectors.h

```diff
@@ -444,6 +444,135 @@ static crypto_test_reference_t aes_gcm_reference[] = {
 	}
 };
 
+static crypto_test_reference_t aes_ccm_reference[] = {
+	/*
+	 * AES-CCM reference from RFC 3610:
+	 * https://tools.ietf.org/html/rfc3610#section-8
```

Comment:
The RFC citation is sufficient. URLs should not be included as they tend to get stale.